backend-api/src/functions/common/session/updateOperations/BiometricTokenIssued/BiometricTokenIssued.ts

```diff
@@ -24,4 +31,49 @@ export class BiometricTokenIssued implements UpdateSessionOperation {
       ":authSessionCreated": { S: SessionState.AUTH_SESSION_CREATED },
     };
   }
+
+  getDynamoDbReturnValues() {
+    return ReturnValue.ALL_NEW;
+  }
+
+  getDynamoDbReturnValuesOnConditionCheckFailure() {
+    return ReturnValuesOnConditionCheckFailure.ALL_OLD;
+  }
+
+  getSessionAttributes(item: Record<string, AttributeValue> | undefined) {
+    return getBaseSessionAttributes(item);
+  }
```

Comment:
The idea for this PR which if I would do in the PR the success txma for this lambda:

An update function can either fail or pass so we would only ever want two shapes of attributes, in this case it would be the base attributes, or attributes after issuing a token (including doc type etc)

So this function would end up looking something like this:

```suggestion
  getSessionAttributes(item: Record<string, AttributeValue> | undefined, isError: boolean) {
  if (isError) {
    return getBaseSessionAttributes(item);
  }
  
  return getSomeOtherSessionAttributes(item);
  }
```

Reply:
The other thought I had is instead of passing something to this function to decide what to update, we could update a state in this class that is then read